src/components/FallbackSignal.tsx

```diff
@@ -0,0 +1,63 @@
+import { FC } from "react"
+import { shallow } from "zustand/shallow"
+
+import clsxm from "@/lib/clsxm"
+
+import Tooltip from "@/components/Tooltip"
+
+import { useQueryStatusStore } from "@/store"
+
+type FallbackSignalProps = {
+  className?: string
+  showTooltip?: boolean
+  showLabels?: boolean
+}
+
+const FallbackSignal: FC<FallbackSignalProps> = ({
+  className,
+  showLabels = true,
+}) => {
+  const [failedQueries] = useQueryStatusStore(
```

Comment:
You can just do: `failedQueries = useQueryStatusStore((state) => state.failedQueries)` since you're only grabbing one thing from the store.

Reply:
right